### DISCUSSION TITLE
Features Request/Suggestion

### DISCUSSION THREAD

**Discussion by MrCypressBai** [WelSimLLC/WelSim-Apps#95] [General]
*  **to change the properties of chart**
    [a] text size of legend & label & axes;
    [b] Color of line;
    [c] etc;
* **Model evaluation**
    [a] SD;
    [b] correlation coefficient；
    [c]  PCCs;
    [d] ...
* **Function icon of Curve Fitting**

**Comment by MrCypressBai** [comment]
Those suggestion are for MatEditor software.

**Comment by lcheng9** [comment]
Thanks for your suggestions. Could you elaborate on what the SD and PCCs are denoted?

**Comment by MrCypressBai** [comment]
Evaluation indexes for curve fitting，e.g. Mean Square, Covariance, Pearson's correlation coefficient, etc..

Case too many model here, and there isn't output button for the data of fitting result.

![image](https://user-images.githubusercontent.com/19180288/192155738-20602703-13fa-46a5-b0e5-f6c1ebf8c48e.png)

**Comment by lcheng9** [comment]
We recently add the controllers for the legend and font size. 
![matEditor_01](https://user-images.githubusercontent.com/106984354/218525547-195a8bd2-cec4-4753-bf81-18a9125c09a5.png)
